VideoWeb/VideoWeb/ClientApp/src/app/security/participant-status.guard.ts

```diff
@@ -34,11 +34,11 @@ export class ParticipantStatusGuard implements CanActivate {
             if (
                 conferenceId &&
                 (!this.router.navigated || urlActive) &&
-                (profile.role === Role.Representative ||
-                    profile.role === Role.Individual ||
-                    profile.role === Role.JudicialOfficeHolder ||
-                    profile.role === Role.QuickLinkObserver ||
-                    profile.role === Role.QuickLinkParticipant)
+                (profile.roles.includes(Role.Representative) ||
```

Comment:
this block of code seems to be in a few places. would it be worth having a const variable with an array of all participant roles so it's easy to ref the one list across the app, incase this list changes